packages/squidex/src/auth.ts

```diff
@@ -41,7 +41,7 @@ export const getAccessTokenFactory = (): (() => Promise<string>) => {
           const jwt = decode<JwtToken>(tk1);
           const currentTime = Date.now() / 1000;
 
-          if (currentTime > jwt.exp) {
+          if (currentTime < jwt.exp) {
```

Comment:
ooof.

Reply:
<=?

Reply:
Do we really care about this one second? :D